db/migrate/20180612164926_add_content_org_id.rb

```diff
@@ -0,0 +1,39 @@
+class AddContentOrgId < ActiveRecord::Migration[5.1]
+  class FakeContent < Katello::Model
+    self.table_name = 'katello_contents'
+    has_many :product_contents, :class_name => 'FakeProductContent', :dependent => :destroy, :foreign_key => 'content_id'
+    has_many :products, :through => :product_contents
+  end
+
+  class FakeProductContent < Katello::Model
+    self.table_name = 'katello_product_contents'
+    belongs_to :product, :class_name => 'Katello::Product', :foreign_key => 'product_id', :inverse_of => :product_contents
+    belongs_to :content, :class_name => 'FakeContent', :foreign_key => 'content_id', :inverse_of => :product_contents
+  end
+
+  class FakeProduct < Katello::Model
+    self.table_name = 'katello_products'
+  end
+
+  def up
+    add_column :katello_contents, :organization_id, :integer, :null => true
+    add_foreign_key :katello_contents, :taxonomies, :column => :organization_id, :primary_key => :id
+
```

Comment:
I wonder if we should wrap this (L22-L32) in an `ActiveRecord::Base.transaction`

Reply:
all migrations are already wrapped in a transaction